backend/src/main/java/com/rota/api/StaffController.java

```diff
@@ -125,23 +125,44 @@ private void verifyManagementRole(String authString) {
     return staffService.getActiveStaff();
   }
 
+  /**
+   * Endpoint to update the information of a staff member.
+   *
+   * @param id           The ID of the staff member to be updated.
+   * @param updatedStaff Updated staff information.
+   * @return The updated {@link Staff} member.
+   */
+  @PutMapping("/staff/$id")
+  @ApiOperation(value = "Lets an authenticated manager update a staff member")
+  public Staff updateStaff(
+      @RequestParam
+      @ApiParam(value = "Staff ID")
+          int id,
+      @RequestBody
+      @ApiParam(value = "Updated staff object")
+          StaffDto updatedStaff
```

Comment:
I think the syntax here is wrong - does it work for you?
I think what you want is
`@PutMapping("/staff/{id}"`
and `int id` should have `@PathVariable(name="id")` instead of `RequestParam`.

Reply:
It seemed to work when I tested it, but I agree that is the correct way of doing it! I'll update now.

Reply:
OK, weird, it gave me 404 every time I tried it.

Reply:
I swear I updated one but oh well! It's correct now so hopefully will work!